packages/datadog-plugin-jest/src/index.js

```diff
@@ -15,7 +15,8 @@ const {
   ERROR_TYPE,
   TEST_PARAMETERS,
   getTestEnvironmentMetadata,
-  getTestParametersString
+  getTestParametersString,
+  getFormattedJestTestParameters
```

Comment:
If this is specific to Jest, why is it needed in the test plugin helpers? I would expect either it should be moved to the Jest plugin, or be made generic so it's not specific to Jest.

Reply:
yeah I thought of that but I wanted to unit test it. That seemed more convenient. I'd move it to `jest` folder and create a different spec file to test it

Reply:
addressed in https://github.com/DataDog/dd-trace-js/pull/1366/commits/a89bfa7d56739356572d9c395e49cb9f6faab2d2. I think it looks nicer now 👍